src/Bridge/Doctrine/Orm/ItemDataProvider.php

```diff
@@ -154,24 +152,42 @@ private function getIdentifiers(string $resourceClass, $id)
      * @param ObjectManager $resourceManager
      * @param string        $resourceClass
      *
+     * @throws RuntimeException
+     *
      * @return \Doctrine\ORM\QueryBuilder
      */
     private function retrieveQueryBuilder(ObjectManager $resourceManager, string $resourceClass)
     {
         $repository = $resourceManager->getRepository($resourceClass);
 
-        if ($repository instanceof  EntityRepository) {
+        if ($repository instanceof EntityRepository) {
             return $repository->createQueryBuilder('o');
         }
 
         if ($resourceManager instanceof EntityManagerInterface) {
```

Comment:
not really because the order matter here:

- try with an `EntityRepository`: optimal case
- try with a `EntityManagerInterface`: other optimal case, but requires to do things differently
- fallback to check if the repository still have this `createQueryBuilder()` method, and assumes it's the right one (technically there is no guarantee here)
- last fallback with the manager before throwing an error